packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/Sidebar.tsx

```diff
@@ -133,10 +133,31 @@ export const Sidebar = ({ showAccounts = true }: SidebarProps) => {
     };
 
     useEffect(() => {
-        const onResize = () => setAutoCollapseSuppressed(false);
-        window.addEventListener('resize', onResize);
+        const resetSuppression = () => {
+            setAutoCollapseSuppressed(false);
 
-        return () => window.removeEventListener('resize', onResize);
+            setTimeout(() => {
```

Comment:
This is weird. Should this be debounced? And shouldn't the setter be only called when the value is not already false?